pkg/minikube/bootstrapper/bsutil/kverify/system_pods.go

```diff
@@ -85,12 +86,30 @@ func ExpectAppsRunning(cs *kubernetes.Clientset, expected []string) error {
 	}
 	klog.Infof("%d kube-system pods found", len(pods.Items))
 
+	for {
+		// core dns deployment has been scaled to 1 pods, wait until there is only 1 pod
+		corednsPods, err := cs.CoreV1().Pods("kube-system").List(context.Background(), meta.ListOptions{
+			LabelSelector: "k8s-app=kube-dns",
+		})
+		if err != nil {
+			return err
+		}
+		if len(corednsPods.Items) == 1 {
```

Comment:
we have a flag that is called disable-optimizations ..or something like that, that would make the coredns 2 replicas
in that case it would be not working